0.6.1/docker-entrypoint.sh

```diff
@@ -61,4 +61,9 @@ elif vault --help "$1" 2>&1 | grep -q "vault $1"; then
     set -- vault "$@"
 fi
 
+# If we are running Vault, make sure it executes as the proper user.
+if [ "$1" = 'vault' ]; then
```

Comment:
If you look in the Consul container (https://github.com/hashicorp/docker-consul/blob/master/0.X/docker-entrypoint.sh#L78) it makes sure permissions are correct if the config dir is bind-mounted in. I think that's probably a good idea -- what do you think?